test/services/composite-keys-manager.test.js

```diff
@@ -2,77 +2,109 @@ import Sequelize, { Op } from 'sequelize';
 import CompositeKeyManager from '../../src/services/composite-keys-manager';
 
 describe('services > composite-keys-manager', () => {
+  const modelBase = {
+    sequelize: { constructor: Sequelize },
+  };
+
   describe('getPrimaryKeyValues', () => {
-    const compositeKeyManager = new CompositeKeyManager();
+    it('should throw for invalid packed key', () => {
```

Comment:
In which case should it throw ? It's missing the context

Reply:
The previous version returned an empty condition when the packed primary key number of elements did not match the actual number of primary keys

=> Calling resource-remover with "1|1" when using a simple primary key would delete the whole collection

This test ensure that it is no longer the case